packages/framework/aqueduct/src/data-objects/pureDataObject.ts

```diff
@@ -108,7 +108,9 @@ export abstract class PureDataObject<I extends DataObjectTypes = DataObjectTypes
 	 * 2. the request url is empty
 	 */
 	public async request(req: IRequest): Promise<IResponse> {
-		return defaultFluidObjectRequestHandler(this, req);
+		return req.url === "" || req.url === "/" || req.url.startsWith("/?")
+			? { mimeType: "fluid/object", status: 200, value: this }
+			: create404Response(req);
 	}
 
 	// #endregion IFluidRouter
```

Comment:
Clean-up later: Remove these `#region` comments

Reply:
Agreed. Pretty sure these are for collapsing areas of code in Visual Studio.